packages/rollup-plugin-html/src/output/getOutputHTML.ts

```diff
@@ -55,6 +55,7 @@ export async function getOutputHTML(params: GetOutputHTMLParams) {
       emittedAssets,
       absolutePathPrefix,
       publicPath: pluginOptions.publicPath,
+      extractAssets: pluginOptions.extractAssets || true,
```

Comment:
Hmm.. could this evaluate to 'true', even when the pluginOption is set to 'false'? All tests passed. New test needed, and maybe test for 'undefined' here?